ui/css/main.css

```diff
@@ -439,19 +360,5 @@ i.fa-plus{
     font-size: 3em;
     text-align: center;
 }
-.car_image img{
-    width: 200px;
-    height: 200px;
-    border-radius: 50%;
-    
-}
-.car_image{
-    width: 100%
-}
 
-/*/* Force table to not be like tables anymore */ 
```

Comment:
I believe this code is of no use any longer. Consider removing it.

Reply:
Please confirm it is not there on the changes made

Reply:
Oh, you removed it in this commit. Sorry about that